content/hi/docs/contribute/analytics.md

```diff
@@ -0,0 +1,25 @@
+---
+title: साइट विश्लेषिकी देखना
+content_type: concept
+weight: 100
+card:
+  name: योगदान
+  weight: 100
+---
+
+<!-- overview -->
+
+इस पृष्ठ में Kubernetes.io विश्लेषिकी डैशबोर्ड के बारे में जानकारी है।
+
+
+<!-- body -->
+
+[डैशबोर्ड देखें](https://datastudio.google.com/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04)।
+
+यह डैशबोर्ड Google Data Studio का उपयोग करके बनाया गया है और Google Analytics का उपयोग करके Kubernetes.io पर एकत्रित जानकारी दिखाता है।
+
+### डैशबोर्ड का उपयोग करना
+
+डिफ़ॉल्ट रूप से, डैशबोर्ड पिछले 30 दिनों के सभी एकत्रित विश्लेषण दिखाता है। विभिन्न दिनांक सीमा मे आने वाला डेटा देखने के लिए date selector का उपयोग करें। अन्य फ़िल्टरिंग विकल्प आपको उपयोगकर्ता का स्थान, साइट तक पहुंचने के लिए उपयोग किए जाने वाले उपकरण, उपयोग किए गए दस्तावेज़ों के अनुवाद, और बहुत से चिज़ो का डेटा देखने की अनुमति देते हैं।
```

Comment:
It should be `चीज़ों` instead of `चिज़ो`.

Reply:
Its done. Thanks for the review